### ISSUE TITLE
Deployment server fails to start when starting the docker-compose cluster

### ISSUE THREAD

**Issue by 772279** [splunk/docker-splunk#383]
I'm trying to simulate the setup of deployment server + 1 universal forwarder so I use the `1deployment1uf.yaml` for my `docker-compose up` command. I'm using it as in 
```shell
docker-compose -f 1deployment1cm.yaml up
```
from the `test_scenarios` directory. 

When starting up, the deployment server shows an error. From the ansible logs,  I see the following.

```
depserver1    |
depserver1    | TASK [splunk_deployment_server : Download remote app] **************************
depserver1    | fatal: [localhost]: FAILED! => {
depserver1    |     "changed": false,
depserver1    |     "dest": "/tmp/",
depserver1    |     "elapsed": 0,
depserver1    |     "gid": 0,
depserver1    |     "group": "root",
depserver1    |     "mode": "01777",
depserver1    |     "owner": "root",
depserver1    |     "response": "HTTP Error 404: Not Found",
depserver1    |     "size": 4096,
depserver1    |     "state": "directory",
depserver1    |     "status_code": 404,
depserver1    |     "uid": 0,
depserver1    |     "url": "http://appserver/splunk_app_example.tgz"
depserver1    | }
depserver1    |
depserver1    | MSG:
depserver1    |
depserver1    | Request failed
depserver1    |
depserver1    | PLAY RECAP *********************************************************************
depserver1    | localhost                  : ok=61   changed=2    unreachable=0    failed=1    skipped=59   rescued=0    ignored=0
depserver1    |
depserver1    | Tuesday 16 June 2020  04:50:58 +0000 (0:00:00.506)       0:00:31.213 **********
depserver1    | ===============================================================================
depserver1    | splunk_common : Get Splunk status --------------------------------------- 6.16s
depserver1    | splunk_common : Start Splunk via CLI ------------------------------------ 4.75s
depserver1    | Gathering Facts --------------------------------------------------------- 1.85s
depserver1    | splunk_common : Update Splunk directory owner --------------------------- 1.64s
depserver1    | splunk_common : Update /opt/splunk/etc ---------------------------------- 0.79s
depserver1    | splunk_deployment_server : Get Splunk status ---------------------------- 0.71s
depserver1    | splunk_common : Get Splunk status --------------------------------------- 0.69s
depserver1    | splunk_deployment_server : Trigger restart ------------------------------ 0.65s
depserver1    | splunk_common : Trigger restart ----------------------------------------- 0.64s
depserver1    | splunk_common : Trigger restart ----------------------------------------- 0.64s
depserver1    | splunk_common : Get Splunk status --------------------------------------- 0.63s
depserver1    | splunk_common : Test basic https endpoint ------------------------------- 0.55s
depserver1    | splunk_deployment_server : Download remote app -------------------------- 0.51s
depserver1    | splunk_common : Wait for splunkd management port ------------------------ 0.45s
depserver1    | splunk_common : Enable splunktcp input ---------------------------------- 0.41s
depserver1    | splunk_common : Find manifests ------------------------------------------ 0.35s
depserver1    | splunk_common : Check for existing installation ------------------------- 0.33s
depserver1    | splunk_common : Cleanup Splunk runtime files ---------------------------- 0.33s
depserver1    | splunk_common : Reset root CA ------------------------------------------- 0.22s
depserver1    | splunk_common : include_tasks ------------------------------------------- 0.21s
depserver1    | splunkd.pid file is unreadable.
depserver1 exited with code 2
```

Is there anyway to setup these docker-compose files ?

**Comment by nwang92** [comment]
This seems to be semi-related to the `appserver` role that gets created as part of the compose: https://github.com/splunk/docker-splunk/blob/develop/test_scenarios/1deployment1so.yaml#L20

The purpose of this container is just to serve the app (splunk_app_example.tgz) at some web address accessible by each of the Splunk instances. It looks like the `tests/fixtures` dir contains the unarchived version of the app (https://github.com/splunk/docker-splunk/tree/develop/tests/fixtures) but not the archived tgz as expected. This is because the test code itself is responsible for creating the tarball: https://github.com/splunk/docker-splunk/blob/develop/tests/test_docker_splunk.py#L2609-L2615

**Comment by nwang92** [comment]
TLDR:
```
cd tests/fixtures
tar czvf splunk_app_example.tgz splunk_app_example
```
then re-run the `docker-compose up` :)
